hwilib/devices/coldcard.py

```diff
@@ -317,7 +317,7 @@ def backup_device(self, label='', passphrase=''):
         result = self.device.download_file(result_len, result_sha, file_number=0)
         filename = time.strftime('backup-%Y%m%d-%H%M.7z')
         open(filename, 'wb').write(result)
-        return {'success': True, 'message': 'The backup has been written to {}'.format(filename)}
```

Comment:
Also losing this?

Reply:
Yes. I couldn't figure out where to put it and keep the api sane, maybe it should be printed to stderr?